README.md

```diff
@@ -1,31 +1,69 @@
 # translator_extensions
+
 The Translator-specific metadata extensions used in the SmartAPI registry
 
 ## x-translator
 #### Status
 This extension's schema has been written, and registry files currently contain this extension.
 
 #### Description
-This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently...
+This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently, there are two (required) properties: component (KP, ARA, ARS, or Utility) and team (one or more Translator team names). Current tags include:
+
+- **infores:** Information Resource CURIE identifying the Translator Resource
```

Comment:
Maybe link the spreadsheet that has the infores stuff, or some details on how to add / modify / delete an infores?